src/lab1/simulator.java

```diff
@@ -4,44 +4,47 @@
 
 // Note that this class is a singleton so we'll use global variables
 public class simulator {
-    // TODO should we care about floating point values?
-    static long t_arrival; // the arrival time of a packet (ms)
-    static long t_departure; // the ddeparture time of a packet (ms)
-    static long t; // elapsed time of simulation aka t in the pseudocode (ms)
-    static long t_start; // start time of simulation, reference point in t (ms)
-    static int lambda; // packets per second
-    static int L; // length of packet in bits
-    static int C; // transmission rate in bits per second
+    static int C;                           // transmission rate in bits per second
+    static boolean is_m_d_one;
+    static int lambda;                      // packets per second
+    static int L;                           // length of packet in bits
     static int num_of_ticks;
-    static int ms_per_second = 1000; // how many milliseconds are in one second
+    // TODO should we care about floating point values?
+    static long t;                          // elapsed time of simulation aka t in the pseudocode (ms)
+    static long t_arrival;                  // the arrival time of a packet (ms)
+    static long t_departure;                // the departure time of a packet (ms)
+    static long t_start;                    // start time of simulation, reference point in t (ms)
+
+    static final int ms_per_second = 1000;  // how many milliseconds are in one second
+
+    // TODO create this class, a superclass of MD1 and MD1K queues
+    static KendallQueue queue;
     static Scanner scanner;
-    static boolean is_m_d_one;
-    static KendallQueue queue; // TODO create this class, a superclass of MD1 and MD1K queues
 
     public static void main(String args[]) {
         initialize_variables();
         t_arrival = calc_arrival_time(); // calculate first packet arrival time
         // TODO appendix B says to repeat this for loop like 5 times or something...
-        for (int i = 0; i <= num_of_ticks; i++) {
+        for (int i = 1; i <= num_of_ticks; i++) {
```

Comment:
Like from the Appendix B of the [manual](https://ece.uwaterloo.ca/~ece358/labs-F16/LAB1_OneQueue.pdf)